shared/modules/gomock_reflect_1203808572/prog.go

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Intended to be merged in?

Reply:
Nope, I'm in the process of removing it from this PR